proposals/new-project-donation-process.md

```diff
@@ -0,0 +1,69 @@
+# KEP-748: Project Donation to Kubeflow
+
+## Summary
+This KEP outlines how community members can donate their project to the Kubeflow community.
+
+## Motivation
+As Kubeflow has become a well established ecosystem and community, several
+projects may want to be donated to the Kubeflow platform to explicitly join our 
+community.
+
+### Goals
+This goal of this process is to give clear guidelines and set expectations 
+for community members about how to be formally included into Kubeflow and 
+the application process.
+
+The decision making process will be separate from the application process and is at the 
+discretion of the Kubeflow Steering Committee. The application and data provided are 
+critical for the KSC to make an informed decision that is best for the longetivity 
+of the project and community.
+
+### Non-Goals
+- Give specific recommendations for evaluating any individual project.
+- Supporting project add-ons.
+
+## Proposal
+The process for donation is intended to be simple but thorough.
+
+Project owners or maintainers will apply for donation by following a four
+step process. 
+
+The process is outlined in six steps:
+
+1. Create a GitHub Issue with a Google Document outlining your proposal (please allow for commentary), the document should have a rough outline with:
+    - Authors
+    - Motivation
+    - Benefits for Kubeflow
+    - Benefits for the Project's Community
+    - Community Metrics
+    - Contributor Metrics
+    - Maintainers
+    - Migration  Plan
+    - Other Related Projects
+2. Submit a Pull Request with the [../proposals/new-project-donation-process.md](application form).
+3. Add your proposal to the Kubeflow Community call to introduce and collect feedback on
+the application.
+4. Work with the Kubeflow Outreach Committee to send an announcement email in `kubeflow-discuss` and publish messages on Slack, LinkedIn, X/Twitter, and other Kubeflow social resources
+5. Schedule meeting with Kubeflow Steering Committee for initial vote and to collect feedback.
+6. Merge or close the Pull Request depending upon the outcome of the final vote.
+
```

Comment:
Should we also say that new project needs to be controlled by one of Kubeflow WG.